c++/test/TestBloomFilter.cc

```diff
@@ -152,9 +152,9 @@ namespace orc {
 
     // test strings
     bloomFilter.reset();
-    const char * emptyStr = u8"";
-    const char * enStr = u8"english";
-    const char * cnStr = u8"中国字";
+    const char * emptyStr = "";
+    const char * enStr = "english";
+    const char * cnStr = "中国字";
```

Comment:
Did you run all tests too?

Reply:
http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2018/p0482r3.html
https://docs.microsoft.com/en-us/cpp/cpp/string-and-character-literals-cpp?view=vs-2019

There was funny warning from clang `type of UTF-8 string literal will change from array of const char to array of const char8_t in C++20`

And your CI said that everything is Ok.

Reply:
As you pointed out, CI doesn't run clang-10.
> You have clang-6 in your CI, but that is too old.